docs/core-configuration.md

```diff
@@ -45,6 +45,7 @@ legacy_version_file = yes
 - `legacy_version_file` - defaults to `no`. If set to yes it will cause plugins that support this feature to read the version files used by other version managers (e.g. `.ruby-version` in the case of Ruby's `rbenv`).
 - `use_release_candidates` - defaults to `no`. If set to yes it will cause the `asdf update` command to upgrade to the latest release candidate release instead of the latest semantic version.
 - `always_keep_download` - defaults to `no`. If set to `yes` it will cause `asdf install` always keep the source code or binary it downloads. If set to `no` the source code or binary downloaded by `asdf install` will be deleted after successful installation.
+- `update_plugins_repository_duration` - defaults is `1440` mins. It will set asdf plugin repository updated duration. When `asdf plugin add`, `asdf plugin list all` command be executed, it will check last update duration to update repository.
```

Comment:
can we name this something like: `plugin_repository_last_check_duration` or `plugin_repository_last_check_duration`?

I am trying to think of something more descriptive but naming this is hard :thinking: